samples/sampler/utils/generateStylesheet.js

```diff
@@ -0,0 +1,66 @@
+import {hexToRGB} from './hexToRGB';
+
+export const generateStylesheet = (componentBackgroundColor, focusBackgroundColor, popupBackgroundColor, textColor, subTextColor) => {
+	const textColorRGB = hexToRGB(textColor);
+	const subTextColorRGB = hexToRGB(subTextColor);
+	const focusBgColorRGB = hexToRGB(focusBackgroundColor);
+	const popupBgColorRGB = hexToRGB(popupBackgroundColor);
+
+	// return stylesheet based on received colors
+	return {
+		// '--sand-bg-color': `${backgroundColor}`, - left it commented because we already have a background global in Storybook
+		'--sand-text-color-rgb': `${textColorRGB}`,
+		'--sand-text-sub-color': `${subTextColor}`,
+		'--sand-shadow-color-rgb': '0, 0, 0',
```

Comment:
If we don't override this value ever, what about to simply remove this? Then, Sandstone's default value would be applied. Isn't it?

Reply:
I want to avoid any static default values in this file as we must check this file also when designed colors are updated. If you have better idea to avoid any human error that miss this file, please feel free to let me know.

Reply:
You are right. Any hardcoded values are applied by Sandstone's default. I removed them and left only the custom colors.

Reply:
👍🏼